packages/react-instantsearch-hooks-dom/src/ui/SearchBox.tsx

```diff
@@ -17,7 +17,8 @@ export type SearchBoxProps = Omit<
   React.HTMLAttributes<HTMLDivElement>,
   'onSubmit' | 'onReset'
 > &
-  Pick<React.HTMLAttributes<HTMLFormElement>, 'onSubmit' | 'onReset'> &
+  Pick<React.HTMLAttributes<HTMLFormElement>, 'onSubmit'> &
+  Required<Pick<React.HTMLAttributes<HTMLFormElement>, 'onReset'>> &
```

Comment:
```suggestion
  Required<Pick<React.HTMLAttributes<HTMLFormElement>, 'onSubmit' | 'onReset'>> &
```

This is simpler, no?

Reply:
We don't require `onSubmit`, it's a forwarded prop but our API doesn't depend on it.